src/codeflare_sdk/ray/rayjobs/rayjob.py

```diff
@@ -17,7 +17,12 @@
 """
 
 import logging
+import os
+import re
+import ast
 from typing import Dict, Any, Optional, Tuple
+from kubernetes import client
+from ...common.kubernetes_cluster.auth import get_api_client
 from python_client.kuberay_job_api import RayjobApi
 
```

Comment:
```suggestion
from python_client.kuberay_cluster_api import RayClusterApi

```

Reply:
GitHub won't let me suggest where I want to for reasons :( On line [148-149](https://github.com/project-codeflare/codeflare-sdk/pull/886/files#diff-a8df90d07b5a68ba7075e6332fa1a2f5ca1cb861693a429075b69e6368081301R148-R149) could we instanciate the cluster api alongside the job api?